builtin/tfc/config_sourcer.go

```diff
@@ -300,6 +303,14 @@ func (cs *ConfigSourcer) read(
 
 				continue
 			}
+			if resp.StatusCode != 200 {
+				L.Error("error in sending request for state version for workspace %q, unexpected response code %q", id, resp.Status)
```

Comment:
Maybe we should log `resp.StatusCode` too here? Not sure if that comes included with `resp.Status`.

Reply:
It's included! Just realized that i'm using string formatting in the logger though - should have included k/v pairs at the end instead. That always gets me 😅

Reply:
Oh yes totally! Yeah update those loggers with the key/value format instead, I forget about that one myself.